fold-and-r.csl

```diff
@@ -140,6 +140,16 @@
           <text variable="URL" prefix=" "/>
         </else>
       </choose>
+      <choose>
+        <if type="speech">
+          <group prefix=", at "  delimiter=", ">
```

Comment:
you don't need the groups here (you never need groups with one element). Just do
```
<group suffix=".">
  <text variable="event" prefix=", at "/>
  <text variable="publisher-pace" prefix=", in "/>
</group>
```
which is also more robust than what you have (in case of a missing place variable).